### ISSUE TITLE
Enforce HTTPS for Matomo requests 🔐

### ISSUE THREAD

**Issue by mpdude** [webfactory/piwik-bundle#33]
It would be great if we could have a configuration setting to enforce all Matomo requests be over HTTPS, no – as it is right now – HTTP or HTTPS depending on the current location.

**Comment by bartfeenstra** [comment]
Please don't. It would hardly amount to anything, while completely preventing anyone from testing over HTTP.

**Comment by mpdude** [comment]
It would only be a configuration setting, of course not forcing everyone using this bundle into using HTTPS (or using HTTPS in all situations). Agree?

**Comment by bartfeenstra** [comment]
Right, I must have misread earlier. My apologies. Enforcing HTTP/HTTPS/either on the server side is a good idea :+1:
